### ISSUE TITLE
Enum type ignores modelsuffix

### ISSUE THREAD

**Issue by mojidabckuu** [yonaskolb/SwagGen#292]
If I add `modelSuffix` option then only objects generated with suffix.
Looking into the code I found this line in Swift code fornatter

```swift
return escapeType("\(modelPrefix)\(name.upperCamelCased())")
```

Shall we add enum suffix support?

**Comment by yonaskolb** [comment]
Yeah I can see value in that. How do you use the modelPrefix? For organisational namespacing, or avoiding collisions?
